content/10.methods.md

```diff
@@ -5,8 +5,7 @@
 From [ISCB's webpage listing **ISCB Distinguished Fellows**](http://web.archive.org/web/20200116150052/https://www.iscb.org/iscb-fellows), we found recipients listed by their full names for the years 2009-2019.
 We gleaned the full name of the Fellow as well as the year in which they received the honor.
 To identify **ISMB Keynote Speakers**, we examined the webpage for each ISMB meeting.
```

Comment:
Do you want to add the clause about the site the reviewer pointed us towards? Was it different than just the ISMB websites?